__test__/index.test.js

```diff
@@ -1,4 +1,4 @@
-const timeout = 15000;
+const timeout = 25000;
 
 describe('Dummy Test ', () => {
   beforeAll(async () => {
```

Comment:
why did you change the timeout  ? you are not written any test here

Reply:
because test failing due to time limit that's why

Reply:
why test is failing any specific reason?

Reply:
because of timeout exceeded